### ISSUE TITLE
Explictly name foreign key column in migration

### ISSUE THREAD

**Issue by cbishopvelti** [bitcrowd/carbonite#93]
My repo is configured with
```
migration_primary_key: [name: :iid, type: :bigserial],
migration_foreign_key: [column: :iid, type: :bigserial]
```
When I try to run the migrations, I get the error:
```
11:07:13.188 [info] create table carbonite_default.changes
** (Postgrex.Error) ERROR 42703 (undefined_column) column "iid" referenced in foreign key constraint does not exist
    (ecto_sql 3.9.2) lib/ecto/adapters/sql.ex:913: Ecto.Adapters.SQL.raise_sql_call_error/1
    (elixir 1.15.4) lib/enum.ex:1693: Enum."-map/2-lists^map/1-1-"/2
    (ecto_sql 3.9.2) lib/ecto/adapters/sql.ex:1005: Ecto.Adapters.SQL.execute_ddl/4
    (ecto_sql 3.9.2) lib/ecto/migration/runner.ex:326: Ecto.Migration.Runner.log_and_execute_ddl/3
    (elixir 1.15.4) lib/enum.ex:1693: Enum."-map/2-lists^map/1-1-"/2
    (elixir 1.15.4) lib/enum.ex:1693: Enum."-map/2-lists^map/1-1-"/2
    (ecto_sql 3.9.2) lib/ecto/migration/runner.ex:289: Ecto.Migration.Runner.perform_operation/3
    (stdlib 5.0.2) timer.erl:270: :timer.tc/2
```

**Comment by maltoe** [comment]
Hey @cbishopvelti ,

thanks for reporting, this is definitely unwanted. Could you do me a favor and try the branch in the PR? :pray:

**Comment by cbishopvelti** [comment]
Yep, that brunch seams to work fine thanks.

**Comment by maltoe** [comment]
Thanks

Released as v0.11.1
